sims/cpp_benchmarks/organize_total_runtimes.py

```diff
@@ -16,13 +16,22 @@
 for i in files:
     ARG = True
     Q = False
-    if re.search('with', i):
-        ARG = False
-    if re.search('queue',i):
-        Q = True
+    SEL = True
+    if re.search('all_neutral', i):
+        SEL = False
+        if re.search('neutral_all_neutral', i):
+            ARG = False
+    else:
+        if re.search('with', i):
+            ARG = False
+        if re.search('queue', i):
```

Comment:
`queue` is whether you ran with parallelization, right/

Reply:
yes.